src/app/root/root.component.ts

```diff
@@ -71,5 +71,9 @@ export class RootComponent implements OnInit {
       .pipe(
         map(([collapsed, mobile]) => collapsed || mobile)
       );
+
+    if (this.router.url === '/500') {
```

Comment:
Perhaps use `getPageInternalServerErrorRoute()` here instead of repeating '/500'?